config/environments/production.php

```diff
@@ -11,6 +11,17 @@
 define('S3_UPLOADS_SECRET', env('S3_UPLOADS_SECRET'));
 define('S3_UPLOADS_REGION', env('S3_UPLOADS_REGION'));
 define('S3_UPLOADS_ENDPOINT', env('S3_UPLOADS_ENDPOINT'));
+define('S3_UPLOADS_BUCKET_URL', env('S3_UPLOADS_BUCKET_URL'));
+
+if (env('S3_UPLOADS_ENDPOINT') && env('S3_UPLOADS_BUCKET_URL') == '') {
+    $s3_endpoint = env('S3_UPLOADS_ENDPOINT');
+    $s3_bucket_name = env('S3_UPLOADS_BUCKET');
```

Comment:
nitpick: why don't you make the variable name the same as the environment variable name? 🤔 
```suggestion
    $s3_uploads_endpoint = env('S3_UPLOADS_ENDPOINT');
    $s3_uploads_bucket = env('S3_UPLOADS_BUCKET');
```

Reply:
`S3_UPLOADS_BUCKET` might be undefined in this condition